ipaserver/plugins/topology.py

```diff
@@ -82,7 +82,12 @@
 
 
 def validate_domain_level(api):
-    current = int(api.Command.domainlevel_get()['result'])
+    current = DOMAIN_LEVEL_0
```

Comment:
Same here, master (4.8) no no supports DL 0 and has DL 1 as minimum level.

Reply:
I added `MIN_DOMAIN_LEVEL` instead of the explicit DL 0.